nautobot_ssot_infoblox/diffsync/adapters/infoblox.py

```diff
@@ -117,9 +119,17 @@ def load_vlans(self):
     def load(self):
         """Load all models by calling other methods."""
         self.load_prefixes()
+        if "prefix" in self.dict():
+            self.job.log(message=f"Loaded {len(self.dict()['prefix'])} prefixes from Infoblox.")
```

Comment:
which log level uses the `log`?

Reply:
Appears to just use Default in my testing. Could change to info?